src/modules/claims/claims.service.ts

```diff
@@ -1438,11 +1441,17 @@ export class ClaimsService {
     const errors: string[] = [];
     const issuerDID = this._signerService.did;
 
-    const proofVerified = await this._verifiableCredentialService.verify(vc);
+    let proofVerified;
+    try {
+      proofVerified = await this._verifiableCredentialService.verify(vc);
+    } catch (e) {
+      proofVerified = false;
+      errors.push((e as Error).message);
+    }
```

Comment:
This change is a bit odd. Did the behaviour of `verifiableCredentialService.verify` change? Or you just noticed a bug?

Reply:
The error for the credential being expired is caught and handled here. If I don't catch it, the whole call with throw an exception.

Reply:
Ah okay, makes sense, thanks.